loopchain/peer/peer_outer_service.py

```diff
@@ -490,10 +490,22 @@ def AnnounceUnconfirmedBlock(self, request, context):
     def BlockSync(self, request, context):
         # Peer To Peer
         channel_name = conf.LOOPCHAIN_DEFAULT_CHANNEL if request.channel == '' else request.channel
+        channel_stub = StubCollection().channel_stubs[channel_name]
+        status_data = asyncio.run_coroutine_threadsafe(
+            channel_stub.async_task().get_status(),
+            self.peer_service.inner_service.loop
+        ).result()
+        if status_data["state"] == "BlockGenerate" and request.block_height > 1:
+            logging.debug(f"Cannot send BlockSync request.height({request.block_height})) during BlockGenerate")
```

Comment:
Please consider: `Cannot reply to BlockSync...` or `Cannot send BlockSyncReply...`